apps/expo/src/app/(dashboard)/account-settings.tsx

```diff
@@ -70,7 +70,7 @@ const otherItems: NavigationListItem[] = [
   {
     title: 'Privacy Policy',
     iconName: 'shield-alt',
-    onPress: () => router.push('/'),
+    onPress: () => router.push('/privacy-policy' as Route<string>),
```

Comment:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/78af60d2-56f0-46bb-84c4-acae433665ec)

change to this so you won't need that

Reply:
Thanks I didn't know you can get around it that way.